lib/cocoapods/installer/user_project_integrator.rb

```diff
@@ -227,7 +227,7 @@ def workspace_path
       # @note   Empty target definitions are ignored.
       #
       def user_project_paths_to_integrate
-        targets_to_integrate.map(&:user_project_path).compact.uniq
+        targets.map(&:user_project_path).compact.uniq
```

Comment:
this method name was renamed generally to explicitly specify the `to_integrate`, can we inline this now? Feels weird to to have to `_integrate` but the full list of targets there.